### PR TITLE
ICU-20857 Update Readme for ICU 66 Preview.

### PR DESCRIPTION
**Pull request by jefgen**
#### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-20857
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

File: icu4j/readme.html [hunk 1]
```diff
@@ -15,20 +15,23 @@
...
+(Last Update: 2019-Nov-27)
 <hr size="2" width="100%">
 
 <p>
 <!-- <b>Note:</b> This is major release of ICU4J. It contains bug fixes and adds implementations
 of inherited API and introduces new API or functionality. -->
+<b>Note:</b> This is a preview release of ICU4J 66.
+The contents of this document may not reflect the recent changes done
+for ICU 66 development. It is not recommended for production use.
 <!-- <b>Note:</b> This is a development milestone of ICU4J 66.
```
Comment (daniel-ju) [inline icu4j/readme.html]: nit: Should we replace the "milestone" comment with "preview" (similar to what you did in the icu4c readme)?

Comment (jefgen) [reply icu4j/readme.html]: Hm, I'm confused. I created a new comment that for "preview" that doesn't mention "milestone" at all.. so I'm not sure what you mean by this?

Comment (daniel-ju) [reply icu4j/readme.html]: Right, in this icu4j file you created a new comment, but in the icu4c file you removed the "milestone" comment and replaced it with the "preview" comment. I was just wondering if the two should be kept consistent.

Comment (jefgen) [reply icu4j/readme.html]: Ah, thanks! Sure, I guess it would make sense to have them both be consistent. I'll push an update in a bit.

**Pull request merged**